### ISSUE TITLE
Can't parse Xbox 360 games with Xenia in .File format

### ISSUE THREAD

**Issue by Joshje100** [doZennn/steam-rom-manager#574]
I have some Xbox 360 games that are in the .File format but I can't parse them

**Comment by lifeandfate1** [comment]
Same issues I just came across due to Xbox Live Arcade Games (Banjo Kazooie & Tooie). 

I worked around it by changing the file extension to a .xex file in Windows, then I added that as a supported file type in the Xbox 360 - Xenia parser.

PARSER SPECIFIC CONFIGURATION
USER'S GLOBS
${title}@(.iso|.ISO|.xex)

Once I was able to get the game to show up I chose my artwork, saved it to Steam, changed the filetype back to FILE by removing the .xex, then finally updated the steam shortcut to remove the .xex on the end of it as well.

Eg. "C:\Games\Emulators\Xbox 360\Xenia\xenia.exe" "C:\Games\Emulators\Xbox 360\Games\Banjo Tooie"

Not a clean way of doing it but if you just want it to show up, that worked. I'm sure someone else has the correct way of getting it to work.
